static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-banner-editorial-independence.js

```diff
@@ -0,0 +1,39 @@
+// @flow strict
+import { makeBannerABTestVariants } from 'common/modules/commercial/contributions-utilities';
+
+const componentType: OphanComponentType = 'ACQUISITIONS_ENGAGEMENT_BANNER';
+const abTestName: string = 'AcquisitionsBannerEditorialIndependence';
+
+const engagementBannerCopyTest: string = `<strong>The Guardian is editorially independent &ndash; our journalism is free from the influence of billionaire owners or politicians. No one edits our editor. No one steers our opinion. </strong> And unlike many others, we haven’t put up a paywall as we want to keep our journalism open and accessible. But the revenue we get from advertising is falling, so we increasingly need our readers to fund our independent, investigative reporting.`;
+
+export const AcquisitionsBannerEditorialIndependence: AcquisitionsABTest = {
+    id: abTestName,
+    campaignId: abTestName,
+    start: '2018-08-06',
+    expiry: '2018-09-06',
```

Comment:
This date does not match the switch expiry...

Just mentioning in case it is unintentional, otherwise ignore me! 😁

Reply:
It wasn't intentional, good catch. I'll make them match.